toolkit/tools/imagegen/configuration/kernelcommandline.go

```diff
@@ -18,6 +18,7 @@ import (
 //   end of the kernel command line
 type KernelCommandLine struct {
 	ImaPolicy        []ImaPolicy `json:"ImaPolicy"`
+	SELinux          SELinux     `json:"SELinux"`
```

Comment:
I realized I missed a bit of code here in the initial patch set. In `func (k *KernelCommandLine) IsValid()` We should be calling `SELinux.IsValid()`. I expect this will cause a bunch of test failures. Let me have a quick poke at it and see what barfs. If it's ugly I'll throw a patch together from my side.

Reply:
Resolved with the patch I sent you.